pkg/skuba/actions/cluster/init/manifests.go

```diff
@@ -874,6 +874,22 @@ spec:
                   fieldPath: spec.nodeName
           command:
             - /usr/bin/kured
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: 8080
+            # The initial delay for the liveness probe is intentionally large to
+            # avoid an endless kill & restart cycle if in the event that the initial
+            # bootstrapping takes longer than expected.
+            initialDelaySeconds: 120
+            failureThreshold: 10
+            periodSeconds: 10
```

Comment:
as the default check for the reboot sentinel is only once per hour, should we also increase this number here?

Reply:
This checks if `kured` service is alive. Reboot check is done by running `kured`. I think it can stay 10 seconds

Reply:
The main goal of having probes there is to check whether kured is able to connect to the apiserver, so IMO having a small timeouts for checks is a good idea.

Reply:
i was just thinking because the upstream kured manifest doesn't have this livenessProbe and seems to work fine without it in most cases. just want to avoid having too many computations (checks every 10s) for nothing

Reply:
> The main goal of having probes there is to check whether kured is able to connect to the apiserver, so IMO having a small timeouts for checks is a good idea.

wouldn't we have to then check against the kube-apiserver/healtz endpoint?

Reply:
> wouldn't we have to then check against the kube-apiserver/healtz endpoint?

We can't monitor kube-apiserver endpoint. Only POD service has to be checked, there might be other reasons that `kured` service doesn't want to start than only problem with reaching Kubernetes API.

Reply:
> i was just thinking because the upstream kured manifest doesn't have this livenessProbe and seems to work fine without it in most cases. just want to avoid having too many computations (checks every 10s) for nothing

I have increased it to 60 seconds, hopefully it will be fine.